samcli/commands/validate/validate.py

```diff
@@ -18,24 +18,26 @@
 
 @click.command("validate",
                short_help="Validate an AWS SAM template.")
+@click.option("--profile",
```

Comment:
Can you abstract this out to how the `@template_option` is done? `--profile` is used across commands and would be nice to have a single decorator to it.

[Here is the template_option](https://github.com/awslabs/aws-sam-cli/blob/develop/samcli/commands/local/cli_common/options.py#L35) that you can use as an example.